unittests/primitives/readlist.lgo

```diff
@@ -4,3 +4,5 @@ show :userInput = rl
 pr count :userInput
 show item 1 :userInput
 show item 2 :userInput
+make "userInput2 readlist
+pr count :userInput2

```

Comment:
Besides the length of the list, should also check every item of the list to make sure they match expected values.